### PR TITLE
feat(zimbra): add auto replies tab and datagrid

### PR DESCRIPTION
**Pull request by stif59100**
ref:MANAGER-15496

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/zimbra-ga
| Bug fix?         |no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15496
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I added auto replies tab and datagrid

### CODE REVIEW

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  untill: string;
+  copyTo: string;
+};
+
+const items: AutoRepliesItem[] = [
+  {
+    id: '1',
+    name: 'to@example1.com',
+    from: '11/02/2023',
+    untill: '11/02/2024',
```
Comment (antonymarion) [inline packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx:26]: 2 l in until ?

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    ),
+    label: '',
+  },
+];
+export function AutoReplies() {
+  const { t } = useTranslation('autoReplies');
+
+  return (
+    <div className="py-6 mt-8">
+      {' '}
```
[the remark this answers is not in the document]
Comment (tristanwagner) [reply packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx:84]: you can also replace this by <Notifications /> and <Outlet /> like in the others tabs

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/__test__/AutoReplies.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+afterEach(() => {
+  vi.clearAllMocks();
+});
+
+describe('Redirections page', () => {
+  it('should display page correctly', () => {
+    const { getByTestId } = render(<AutoReplies />);
+    const button = getByTestId('header-until');
+
+    expect(button).toHaveTextContent('zimbra_auto_replies_untill');
```
Comment (antonymarion) [inline packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/__test__/AutoReplies.spec.tsx:29]: untill still 2 l here

Comment (antonymarion) [reply packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/__test__/AutoReplies.spec.tsx:29]: => test failing because of typo
image

**Review by tristanwagner — APPROVED** [review]

**Pull request merged**